src/main/java/at/ac/tuwien/kr/alpha/solver/Antecedent.java

```diff
@@ -0,0 +1,17 @@
+package at.ac.tuwien.kr.alpha.solver;
+
+/**
+ * An interface to reasons of implications as used internally by the solver. This is a lightweight {@link at.ac.tuwien.kr.alpha.common.NoGood} that only
+ * provides an array of literals (in some order) and has an activity that may change.
+ *
+ * Copyright (c) 2019, the Alpha Team.
+ */
+public interface Antecedent {
+
+	int[] getReasonLiterals();
+
+	void bumpActivity();
```

Comment:
Did you consider naming this `increaseActivity`? I am asking because it appears that `bumpActivity` interplays with `decreaseActivity` and I expect some kind of dualism. Of that's the case then increase vs. decrease might highlight this better.

Reply:
The names are inspired by names used in other solvers. `bumpActivity` increases by a constant while `decreaseActivity` decreases by a factor. So they indeed are dual in some way, but also different.